src/Microsoft.VisualStudio.LanguageServices.Razor/DefaultTemplateEngineFactoryService.cs

```diff
@@ -70,8 +73,36 @@ public override RazorTemplateEngine Create(string projectPath, Action<IRazorEngi
 
         private Version GetMvcVersion(string projectPath)
         {
-            // TODO: Need to figure out the actual referenced Mvc version in the project. https://github.com/aspnet/Razor/issues/1183
+            var workspace = _services.WorkspaceServices.Workspace;
+
+            var project = workspace.CurrentSolution.Projects.FirstOrDefault(p =>
```

Comment:
We can also think about just making an API that takes the `Project` and transition to that.